docs/ldml/tr35-general.md

```diff
@@ -945,10 +949,26 @@ The formal syntax for identifiers is provided below.
 
 <tr><td>dimensionality_prefix</td><td>:=</td>
     <td>"square-"<p>| "cubic-"<p>| "pow" ([2-9]|1[0-5]) "-"
-        <ul><li><em>Note:</em> "pow2-" and "pow3-" canonicalize to "square-" and "cubic-"</li></ul></td></tr>
+        <ul>
+			<li><em>Note:</em> "pow2-" and "pow3-" canonicalize to "square-" and "cubic-"</li>
+			<li><em>Note:</em>These are values in &lt;unitIdComponent type=”power”&gt;</li>
+		</ul></td></tr>
 
+<tr><td>simple_unit</td><td>:=</td>
+    <td>prefixed_unit<br/>
+		| (prefix_component "-")* base_component ("-" suffix_component)*<br/>
```

Comment:
I think this should reference prefixed_unit instead of base_component

Reply:
It is more restrictive than what we had before. 
That is taking advantage of the fact that the metric units (roughly, units that can have SI or binary prefixes attached to them) don't have prefix_components or suffix_components.
It would mean that while before you could say "3 megaquarts-imperial" and with this change you can't.
However, thinking about it now, a full solution would require more changes, so I'll fix this.

Reply:
done